doc/whats_new/v1.1.rst

```diff
@@ -217,6 +217,11 @@ Changelog
   `fit` instead of `__init__`.
   :pr:`21567` by :user:`Maggie Chege <MaggieChege>`.
 
+- |FIX| :class:`decomposition._base` more safely calculates precision using
```

Comment:
Since this is public facing, it's best to list out all the estimators that are affected by this change.